submitted_models/marble_spot_sensor_config_2/model.config

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0"?>
+<model>
+  <name>marble_spot_sensor_config_2</name>
+  <sdf version="1.6">model.sdf</sdf>
+  <description>
+    Boston Dynamics Spot of CMarble team.
```

Comment:
Should this be Marble?

Reply:
Fixed in https://github.com/osrf/subt/commit/7349e416ef7ccac9a4c3775ecf3057d6e17eef8d